packages/block-library/src/reset.scss

```diff
@@ -98,4 +98,40 @@
 		line-height: revert;
 		font-weight: revert;
 	}
+
+	select,
+	input,
+	textarea,
+	button {
+		font-family: system-ui;
+		all: revert;
```

Comment:
Just for the public record :) `all: revert;` is also reverting [`-webkit-user-modify`](https://developer.mozilla.org/en-US/docs/Web/CSS/user-modify) to `read-only` preventing `contenteditable` from working.